### PR TITLE
Bugfix/demos polishing

### PR DESCRIPTION
**Pull request by anton-karlovskiy**
@addyosmani 

Mentioned demos are updated.
`cra-device-class-aware-loading` is renamed as `cra-device-class-aware-code-splitting`
It's ready for review.

### CODE REVIEW

File: node-network-aware-loading/server.js [hunk 1]
```diff
@@ -17,7 +17,8 @@
 const express = require('express');
 const path = require('path');
 const cors = require('cors');
-const request = require('request');
+
+const IMAGES_PATH = 'public/assets/images';
```
Comment (addyosmani) [inline node-network-aware-loading/server.js]: It looks like there's something a little strange happening with node-network-aware-loading. I've tried freshly installing all dependencies, building and starting but am failing to get the images served in dev or prod starts:

image

Occasionally it seems to be missing references to dependencies, but will let you dig in deeper:

```
ReferenceError: fs is not defined
    at /Users/addyo/projects/adaptive-loading/node-network-aware-loading/server.js:68:22
```

File: cna-memory-considerate-animation/components/Layout/index.js [hunk 2]
```diff
@@ -31,13 +31,8 @@ const Layout = ({ children, clientHintDeviceMemory, hookMemoryStatus }) => {
       <Head>
         <title>Adaptive Animation</title>
```
Comment (addyosmani) [inline cna-memory-considerate-animation/components/Layout/index.js]: CNA: GitHub link in this demo should link back to the original implementation we adapted (right now it links to your personal GitHub :))

image

Comment (anton-karlovskiy) [reply cna-memory-considerate-animation/components/Layout/index.js]: Let me change the link to the original implementation.
Should we link to GitHub repo or deployed URL?

File: node-network-aware-loading/server.js (hunk 1 above)
Comment (anton-karlovskiy) [reply node-network-aware-loading/server.js]: Sorry, `const fs = require('fs');` was missing. I fixed it.

File: cna-memory-considerate-animation/components/Layout/index.js (hunk 2 above)
Comment (addyosmani) [reply cna-memory-considerate-animation/components/Layout/index.js]: I think we should link to the repo

Comment (anton-karlovskiy) [reply cna-memory-considerate-animation/components/Layout/index.js]: It's done so.

Comment (anton-karlovskiy) [reply cna-memory-considerate-animation/components/Layout/index.js]: Please check it.

**Comment by addyosmani** [comment]
After reviewing the other changes, I think this is safe to land. My one ask after it does is if we can double check all the deployed URLs function the way they did before (ran into some minor build issues locally that I don't believe are related to your changes).

**Comment by anton-karlovskiy** [comment]
> @anton-karlovskiy After reviewing the other changes, I think this is safe to land. My one ask after it does is if we can double check all the deployed URLs function the way they did before (ran into some minor build issues locally that I don't believe are related to your changes). Sound good?

Sure, after this Polish was completed, I double checked all deployed URLs. They were all working as before.
The `node-network-aware-loading` was broken only locally, as I updated the online assets to local ones and missed importing `fs`. But they were already using local assets in firebase functions. So not broken for deployed URL.

Let me double check.

**Pull request merged**